soh/src/overlays/actors/ovl_Fishing/z_fishing.c

```diff
@@ -5174,6 +5174,38 @@ static Vec3s sSinkingLureLocationPos[] = {
     { 553, -48, -508 },
 };
 
+//Enhancement: Quit Fishing At Door
+void Fishing_QuitAtDoor(Fishing* this, PlayState* play) {
+    if ((Message_GetState(&play->msgCtx) == TEXT_STATE_CHOICE) && Message_ShouldAdvance(play)) {
+        Message_CloseTextbox(play);
+
+        switch (play->msgCtx.choiceIndex) {
+            case 0:
+                if (D_80B7E084 == 0) {
+                    Message_ContinueTextbox(play, 0x4085);
+                } else if (sLinkAge == 1) {
+                    Message_ContinueTextbox(play, 0x4092);
+                }
+                if (play) {}
```

Comment:
what does this line do?

Reply:
As far as I can tell, nothing - as you might expect from just looking at it. It's coming directly from line 5124, where the normal "take rod" is handled. Doesn't appear to change anything at all when it's commented out in vanilla path or the enhancement path. That being said, there are lots of pieces of the project I still don't understand, so I didn't want to risk cutting it.

Reply:
Makes sense; it has been yeeted